graylog2-web-interface/src/views/components/widgets/WidgetFilterMenu.jsx

```diff
@@ -2,9 +2,6 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Button, OverlayTrigger, Popover } from 'react-bootstrap';
 
-import connect from 'stores/connect';
-
-import { SearchStore } from 'views/stores/SearchStore';
 import QueryInput from '../searchbar/AsyncQueryInput';
```

Comment:
Should this import as `AsyncQueryInput` ?

Reply:
And, conversely, should the `AsyncQueryInput.jsx` export the var named `AsyncQueryInput`?

Reply:
This is a special case. There are two files, `QueryInput.jsx`, which is just the component, `AsyncQueryInput.jsx`, which is reexporting the same component, but in an async way using `loadable-components`, so dependencies like `react-ace` are splitted out and loaded asynchronously. So from my perspective the export is fine, importing it as `QueryInput` too, because it _is_ that component when rendering.